docs/gaia-tutorials/join-mainnet.md

```diff
@@ -4,6 +4,12 @@ order: 3
 
 # Join the Cosmos Hub Mainnet
 
+> Note: The current mainnet (`cosmoshub-4`) has performed the first [in-place store migration](https://docs.cosmos.network/v0.43/architecture/adr-041-in-place-store-migrations.html#adr-041-in-place-store-migrations) upgrade. That means that an [upgrade proposal](https://wallet.keplr.app/#/cosmoshub/governance?detailId=51) was passed to start using the [v5.0.0]() release of the gaia node. As a result the [v4.2.6](https://github.com/cosmos/gaia/releases/tag/v4.2.6) version of the gaia node would panic and stop at block [6910000](https://github.com/cosmos/gaia/blob/main/docs/migration/cosmoshub-4-delta-upgrade.md#Upgrade-will-take-place-July-12,-2021). At that point node operators installed the [v5.0.2](https://github.com/cosmos/gaia/releases/tag/v5.0.2) version of the gaia node and then started the node again. This type of upgrade preserves the same `chain-id` but is otherwise similar to a traditional hub upgrade, meaning historical state is lost after the upgrade takes place (still accessible of course by a v4.6.2 full node).
```

Comment:
```suggestion
> Note: The current mainnet (`cosmoshub-4`) has performed the first [in-place store migration](https://docs.cosmos.network/v0.43/architecture/adr-041-in-place-store-migrations.html#adr-041-in-place-store-migrations) upgrade. That means that an [upgrade proposal](https://wallet.keplr.app/#/cosmoshub/governance?detailId=51) was passed to start using the [v5.0.0]() release of the gaia node. As a result the [v4.2.6](https://github.com/cosmos/gaia/releases/tag/v4.2.6) version of the gaia node would panic and stop at block [6910000](https://github.com/cosmos/gaia/blob/main/docs/migration/cosmoshub-4-delta-upgrade.md#Upgrade-will-take-place-July-12,-2021). At that point node operators installed the [v5.0.2](https://github.com/cosmos/gaia/releases/tag/v5.0.2) version of the gaia node and then started the node again. This type of upgrade preserves the same `chain-id` but is otherwise similar to a traditional hub upgrade, meaning historical state can't be queried—however the historical state is preserved which means you can run a `v4.2.6` binary on the same machine to make historical queries.
```

Reply:
@alexanderbez is this accurate now? could you run two binaries on the same machine and use one to make historical queries and the other for queries after the upgrade height?

Reply:
Yup! You certainly can :)